util/expand-liquid.js

```diff
@@ -0,0 +1,53 @@
+/***
+ * For local sample development / use without the Shopify CLI only.
+ * Shopify CLI will expand liquid template files when these templates are used.
+ */
```

Comment:
The wording here isn't clear to me. Am I correct in thinking that I should use this script when I'm not using the Shopify CLI?

Reply:
Yeah that's right. So how this liquid expand thing works is if you use the CLI command to get a copy of this code 

````
yarn create @shopify/app --template https://github.com/Shopify/function-examples/sample-apps/discounts-tutorial
````

This will take the liquid files and fill in the `app_name` and just give you the proper `.js` or `.graphql` file. 

However, this utility script is if you don't want to use that command for some reason and just want to clone this repo directly and work from this repo. Does that make sense?